packages/types/src/stackframe.ts

```diff
@@ -15,4 +15,5 @@ export interface StackFrame {
   addr_mode?: string;
   vars?: { [key: string]: any };
   debug_id?: string;
+  module_metadata?: object;
```

Comment:
Wondering if we should type this as `unknown` 🤔

Reply:
Oh I missed that type! Maybe `unknown`?